homeassistant/components/climate/__init__.py

```diff
@@ -43,6 +43,9 @@
 STATE_AUTO = "auto"
 STATE_DRY = "dry"
 STATE_FAN_ONLY = "fan_only"
+STATE_AWAY_HOLD = "away"
+STATE_HOME_HOLD = "home"
+STATE_TEMPERATURE_HOLD = "temp"
```

Comment:
Are these really states? At least for some of them it seems the device will still be in heat or cold state at the same time as using one of these modes. They are meta states sort of.